unity/Assets/Maroon/SharedEntities/Player/ModeFirstPerson/ModeFirstPerson.cs

```diff
@@ -123,6 +125,10 @@ void Start()
         // force the crouch state to false when starting
         SetCrouchingState(false, true);
         UpdateCharacterHeight(true);
+
+        // Wait for SoundManager to be instantiated, then get its soundEffect AudioSource
+        yield return new WaitUntil(() => SoundManager.Instance != null);
+        audioSource = SoundManager.Instance.soundEffectSource;
```

Comment:
Get the audioSource component from global SoundManager. Just in case, wait for it to be instantiated first.